doc/source/index.rst

```diff
@@ -33,4 +33,11 @@ Packages
       :text-align: center
       :class-title: pyansys-card-title
 
-      Pythonic interface to Granta MI RecordLists services.
\ No newline at end of file
+      Pythonic interface to Granta MI RecordLists services.
+
+    .. grid-item-card:: PyGranta JobQueue: documentation
+      :link: https://jobqueue.grantami.docs.pyansys.com/
```

Comment:
Does this want to point at latest explicitely?

Reply:
The others don't. I'd say we should be consistent in this PR, and if we want to update them all we can do that separately.

Is there any benefit to pointing to latest explicitly? Might help the SEO?

Reply:
I have no idea about SEO, but keeping it consistent sounds as good a reason as any to leave it